kubernetes/e2e_test/test_client.py

```diff
@@ -65,6 +72,23 @@ def test_pod_apis(self):
 
         name = 'busybox-test-' + short_uuid()
         pod_manifest = manifest_with_command(name, "while true;do date;sleep 5; done")
+
+        # wait for the default service account to be created
```

Comment:
better to give a timeout than wait forever

Reply:
done

/hold
wait for test results

Reply:
/hold cancel
The test is green